chore: Add Node.js engine requirement (>=20.19.0) to package.json after previous node update

Friftycode opened the pull request:
### **Description**
Added Node.js engine requirement (>=20.19.0) to package.json

NghiNg reviewed (commented):
It sounds like a good update, but the commit changes does not have any updates to package.json, how does this work? Did you forget to commit a file?

Friftycode commented:
> It sounds like a good update, but the commit changes does not have any updates to package.json, how does this work? Did you forget to commit a file?

A week or two ago, I updated Node and added the engines field to package.json. However, after pulling the latest changes from the main branch in the IDE, I noticed that the engine requirement still showed up as an uncommitted change, meaning it hadn't been included in any previous commit.

To address this, I created a new PR specifically to add the engine check:
```json
  },
  "engines": {
    "node": ">=20.19.0"
  }
}
```

NghiNg reviewed (approved):
Approve regarding previous comment thread. Just check my comment in the thread.
